api/config/config_local.json

```diff
@@ -3,6 +3,7 @@
   "firecloud": {
     "billingAccountId": "014D91-FCB792-33D2C0",
     "billingProjectPrefix": "aou-test-f1-",
+    "clusterMaxAgeDays": 10,
```

Comment:
Could probably be 5 like test? Any reason to make this 10?

Reply:
I could also picture this as 7 or something, just to keep it on the same scale as stable. (Also, a week seems like a reasonable time?)

Either way, I second Dan that I think it should be about the same time as test. Although I see what you're going for, since development is likely happening locally, and thus we are likely to be reusing clusters. I suspect that five minutes every five days is acceptable cost though.